docs/building_lbann.rst

```diff
@@ -226,7 +226,6 @@ Hydrogen, and LBANN separately, by whatever means they choose.
         spack install
         spack env loads # Spack creates a file named loads that has all of the correct modules
         source loads
```

Comment:
Before this line, we should add `source ${SPACK_ROOT}/share/spack/setup-env.sh` because clean installations of spack (i.e., those with no modules in them) will not modify `MODULEPATH`. Running the setup script again should not have negative side effects for anyone and it will be necessary for new users who don't have any spack-installed packages.

Reply:
Good insight.